zeroconf/__init__.py

```diff
@@ -882,21 +882,15 @@ class State(enum.Enum):
         init = 0
         finished = 1
 
+    @staticmethod
+    def is_type_unique(type_: int) -> bool:
+        return type_ == _TYPE_TXT or type_ == _TYPE_SRV or type_ == _TYPE_A or type_ == _TYPE_AAAA
+
     def add_question(self, record: DNSQuestion) -> None:
         """Adds a question"""
         self.questions.append(record)
 
     def add_answer(self, inp: DNSIncoming, record: DNSRecord) -> None:
-
-        """Only support for unique answers"""
-        if (
-            record.type == _TYPE_TXT
-            or record.type == _TYPE_SRV
-            or record.type == _TYPE_A
-            or record.type == _TYPE_AAAA
-        ):
-            assert record.unique
-
```

Comment:
Do we not want this anymore?

Reply:
I noticed when refactoring that it wasn't needed as the same assertion is called inside the called function add_answer_at_time()